bruno/collections/Rafiki/Examples/Web Monetization/Continuation Request.bru

```diff
@@ -0,0 +1,33 @@
+meta {
```

Comment:
I'm trying to think whether we need a separate folder for this. Technically, we can get a large grant, and then keep re-running the same "Create Outgoing Payment" request. Maybe it's good to have, however, just thinking out loud

Reply:
> I'm trying to think whether we need a separate folder for this

In other words, maybe not adding this new "Web Monetization" example? 

I mentioned my reasoning a bit in this comment (while questioning the name) https://github.com/interledger/rafiki/pull/2753#discussion_r1625241575. Basically I didn't find it trivial to alter the existing Open Payments one to do this. It took me some time to put together and I dont even remember all the changes off the top of my head (I would not want to have to re-figure it out). Feel like it could save myself and other some time.